crawlerr/config.py

```
@@ -14,91 +13,118 @@ class CrawlerrSettings(BaseSettings):
+    # Reranker Configuration
+    reranker_model: str = Field(default="tomaarsen/Qwen3-Reranker-0.6B-seq-cls", alias="RERANKER_MODEL")
+    reranker_enabled: bool = Field(default=True, alias="RERANKER_ENABLED")
+    reranker_top_k: int = Field(default=10, alias="RERANKER_TOP_K", gt=0, le=100)
+    reranker_max_length: int = Field(default=512, alias="RERANKER_MAX_LENGTH", gt=0, le=4096)
+    reranker_fallback_to_custom: bool = Field(default=True, alias="RERANKER_FALLBACK_TO_CUSTOM")
+    
+    @field_validator("reranker_model", mode="before")
+    @classmethod
+    def _validate_reranker_model(cls, v: str) -> str:
+        if not v or not str(v).strip():
+            raise ValueError("RERANKER_MODEL must be a non-empty string")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)